proposed-level-2-roadmap.md

```diff
@@ -85,6 +90,10 @@ enough value in based on the rumblings I've heard from Mozilla to be worth imple
     removing it entirely, perhaps compromising based on use cases by moving it out of the address
     bar into developer tooling or extensions.
 
+5.  User agents should treat insecure downloads initiated from secure contexts as blockable mixed
+    content. To limit breakage, this measure may be restricted to high-risk downloads, e.g.,
+    executable file types.
```

Comment:
If we're restricting when we block, is that really just saying we want it to be optionally blockable?